packages/docs/docs/website/versioned_docs/version-2.0.0/testing.md

```diff
@@ -111,6 +111,8 @@ This means that calls to contracts with more than one function named `initialize
 The current solution to this issue is to `npm install zos-lib` and use the same helper function `zos create` uses: `encodeCall`. `encodeCall` receives the signature of your `initialize` function, as well as its arguments and their types. It then crafts the calldata which you can send in a raw call. Here's an example:
 
 ```js
+const encodeCall = require('zos-lib/lib/helpers/encodeCall').default;
```

Comment:
shouldn't this be `const { encodeCall } = require('zos-lib')` ?

@jcarpanelli can you confirm if this change should be placed in the versioned dir?

Reply:
This should be added in both `docs/docs/website/versioned_docs/version-2.0.0/testing.md` and `docs/docs/docs/testing.md` so we don't lose track of it in future versions of this file.

Reply:
I've just added it to `docs/docs/docs/testing.md` too!